platforms/corda-ent/charts/float/files/firewall.conf

```diff
@@ -0,0 +1,37 @@
+firewallMode: FloatOuter
+
+# Uses /opt/corda/workspace as base directory
+networkParametersPath: "network-parameters"
+
+inboundConfig: {
+  listeningAddress: "{{ .Values.dmz.external}}:{{ .Values.node.p2pPort}}"
+}
+
+floatOuterConfig: {
+    floatAddress: "{{ .Values.dmz.internal}}:{{ .Values.bridge.tunnelPort }}"
+    expectedCertificateSubject: "{{ .Values.bridge.legalName }}"
+    tunnelSSLConfiguration: {
+           keyStorePassword: "password"
+           trustStorePassword: "password"
+           sslKeystore: "certificates/float.jks"
+           trustStoreFile: "certificates/trust.jks"
+    }
+}
+
+revocationConfig: { mode: "EXTERNAL_SOURCE"})
+
+certificatesDirectory: "certificates"
+sslKeystore: "certificates/sslkeystore.jks"
+trustStoreFile: "certificates/truststore.jks"
+keyStorePassword: "password"
+trustStorePassword: "password"
+silencedIPs: []
+
+enableAMQPPacketTrace: false
+artemisReconnectionIntervalMin: 5000
+artemisReconnectionIntervalMax: 60000
+politeShutdownPeriod: 1000
+p2pConfirmationWindowSize: 1048576
+auditServiceConfiguration: {
+  loggingIntervalSec: 60
+}
```

Comment:
Add newline

Reply:
Fixed in [9862bdb](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/893/commits/9862bdbebd85ac9508aaec96a1807981550cf571)